cmd/gateway/commands.go

```diff
@@ -285,6 +312,34 @@ func createStaticModeCommand() *cobra.Command {
 		"Use NGINX Plus",
 	)
 
+	cmd.Flags().Var(
+		&usageReportSecretName,
+		usageReportSecretFlag,
+		"The namespace/name of the Secret containing the credentials for NGINX Plus usage reporting.",
+	)
+
+	cmd.Flags().Var(
+		&usageReportServerURL,
+		usageReportServerURLFlag,
+		"The base server URL of the NGINX Plus usage reporting server.",
+	)
+
+	cmd.MarkFlagsRequiredTogether(usageReportSecretFlag, usageReportServerURLFlag)
+
+	cmd.Flags().Var(
+		&usageReportClusterName,
+		usageReportClusterNameFlag,
+		"The display name of the Kubernetes cluster in the NGINX Plus usage reporting server.",
+	)
+
+	cmd.Flags().BoolVar(
+		&usageReportSkipVerify,
+		usageReportSkipVerifyFlag,
+		false,
+		"Disable client verification of the NGINX Plus usage reporting server certificate. "+
```

Comment:
The existing cluster connector appears to assume http or insecure https. I made the same assumption. I can remove the comment about "not to be used in production", because maybe the user will be in a situation where this is not important.

Reply:
I suggest we re-evaluate this requirements with product management, it might be an omission. It is hard to reconcile we don't support TLS where the NGF can trust the NIM.

Reply:
discussed this online. the product management is ok not supporting configuring CA.